### PR TITLE
Editorial updates to Smstateen

### PR DESCRIPTION
**Pull request by ved-rivos**

### CODE REVIEW

**Review by aswaterman — APPROVED** [review]

File: src/priv-csrs.adoc [hunk 1]
```diff
@@ -282,8 +282,26 @@ Supervisor count overflow.
 
 |`0x5A8` |SRW |`scontext` |Supervisor-mode context register.
 
-4+^|Supervisor Resource Management Configuration
-|`0x181` |SRW |`srmcfg` |Supervisor Resource Management Configuration.
+//4+^|Supervisor Resource Management Configuration
+//|`0x181` |SRW |`srmcfg` |Supervisor Resource Management Configuration.
```
Comment (aswaterman) [inline src/priv-csrs.adoc:286]: @ved-rivos Any reason why this is commented out? Should I just make a PR to undo this change?

Comment (aswaterman) [reply src/priv-csrs.adoc:286]: https://github.com/riscv/riscv-isa-manual/issues/2027

Comment (ved-rivos) [reply src/priv-csrs.adoc:286]: @aswaterman - it was commented out waiting for Priv 1.13 to be released. It can be uncommented now.

Comment (aswaterman) [reply src/priv-csrs.adoc:286]: Thanks, I'll handle it.
